### DISCUSSION TITLE
4.3v mA capacity - brake light 38.5 mA relay

### DISCUSSION THREAD

**Discussion by dhackney** [danielnilsson9/bbs-fw#168] [Q&A]
I'd like to use a DPST relay powered by the 4.3v on the shift sensor cutoff circuit to control both a brake light and the brake cutoff. 

The relay is rated for 47.6 mA on the coil. 

I have identified the TI LM5007 buck converter on the controller.

I do not know if the TI LM5007 is providing 5v or 4.3v.

I have been unable to identify another DC to DC voltage converter that provides either 5v or 4.3v in order to determine its capacity. 

Does anyone know the component providing 4.3v and/or its mA capacity? 

Thanks

(Edited to correct relay mA rating and shift cutoff)

**Comment by aalm** [comment]
You might want to add which motor(=controller) you have, as they're not the same,
and do have differences, while those might be relatively small in this area.

fwiw., there is some work-in-progress(#160) for using light output(6V) for the brakelight.

**Comment by dhackney** [comment]
@aalm thanks for the quick reply.


RE: motor/controller

The controller is BBSHD


RE: light output (6V)

The brake light is powered by the headlight, so I only need to power the relay coil from the brake cutoff.

The headlight is powered directly by the battery (52v).

**Comment by dhackney** [comment]
Shift cutoff BBSHD motor controller connection point:

![BBSHD-controller-shift-cutoff-red](https://github.com/danielnilsson9/bbs-fw/assets/1709662/ee6a3d44-302d-4ae7-9118-79bc9d870fcf)
